### ISSUE TITLE
ugof calculation: correlation between number of onsets and ugof

### ISSUE THREAD

**Issue by Jellevanderwerff** [Jellevanderwerff/thebeat#50]
MWE

```
import thebeat
import numpy as np

seq_iois = np.random.default_rng(123).normal(500, 50, 100)
n_onsets = range(1, len(seq_iois) + 2)
ugofs = []

# get ugofs for increasing number of onsets
for n in n_onsets:
    seq = thebeat.Sequence(seq_iois[:n])
    ugofs.append(thebeat.stats.get_ugof_isochronous(seq, 500))

# calculate correlation between number of onsets and ugof
corr = np.corrcoef(n_onsets, ugofs)[0, 1]
print(corr)  # output: 0.9352718532998031
```

**Comment by Jellevanderwerff** [comment]
I think this is serious enough to warrant a bug fix

**Comment by Jellevanderwerff** [comment]
Now I'm not convinced it's a _thebeat_ issue. Will look into it more, later.

**Comment by Jellevanderwerff** [comment]
Though, still, if both the test onsets and ref onsets start with zero, that value should be discarded. But have to think about the best way to do that. I think making a new array of onsets based on the IOIs is the easiest. That way, we also take into account sequences that do not have the first onset at zero.

**Comment by Jellevanderwerff** [comment]
For the comment directly above, this was fixed in ff43c82. The correlation seems to be a property of the ugof (it's the same for the nPVI).
